spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -178,6 +178,96 @@ public void testSave() throws Exception {
 	}
 
 	@Test
+	public void testFindRelatedStreams() throws Exception {
```

Comment:
Since the `actual` changes are in the `controller` side I would like to know what advantage we get by testing this without web based unit tests (where we verify the usecases).